### ISSUE TITLE
Talar tilt test positive

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11152]
**Preferred term label:**
Talar tilt test positive

**Synonyms**

**Definition (free text, please give PubMed ID)**
>15 degrees difference in ankle motion with manual inversion pressure on the injured compared to the contralateral healthy ankle

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Ankle sprain

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001317
- label: Posititve talar tilt test
- parents: [Abnormal ankle physiology (HP:0034674)](https://hpo.jax.org/app/browse/term/HP:0034674)
- pmid: PMID:24995627|PMID:33543095
- def: An abnormal result of the talar tilt test. This test assesses ankle ligament integrity, particularly the calcaneofibular ligament (CFL) and the deltoid ligaments. To perform the test, the foot is placed at an angle of 90 degrees with the leg, the calcaneofibular ligament is perpendicular and the anterior talofibular ligament is parallel to the long axis of the talus. Therefore, inversion in this position results in strain on the calcaneofibular ligament.
- comment: Post-traumatic laxity is defined as excessive joint range of motion beyond normal physiologic range of motion.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11152
- synonyms:
